pcsx2-qt/Settings/ControllerGlobalSettingsWidget.ui

```diff
@@ -122,7 +122,7 @@
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
-         <string>The XInput source provides support for XBox 360 / XBox One / XBox Series controllers, and third party controllers which implement the XInput protocol.</string>
+         <string>The XInput source provides support for Xbox 360 / Xbox One / Xbox Series controllers, and third party controllers which implement the XInput protocol.</string>
```

Comment:
Perhaps the text is better to say that looking at SDL first is better. Just as QoL as xbox works fine with SDL.

Reply:
Up to you guys, maybe out-of-scope for this PR.